app/controllers/sign_up/email_resend_controller.rb

```diff
@@ -22,13 +22,13 @@ def create
 
     private
 
-    def downcased_email
-      params[:resend_email_confirmation_form][:email].downcase
+    def formatted_email
+      params[:resend_email_confirmation_form][:email].downcase.strip
```

Comment:
I see that we are calling `.downcase.strip` on params in at least two places. May be overkill, but could be nice to create a little presenter class that does this so we can test it in isolation.